src/admin/components/dashboard/Dashboard_map.tsx

```diff
@@ -1,9 +1,7 @@
 import { useEffect, useState } from "react";
 import { GoogleMap, useLoadScript, Marker } from "@react-google-maps/api";
-import { IOnlineDriver } from "../../../api/type";
-import { OnlineDriverApi } from "../../../api/OnlineDriverApi";
-import { OnlineDriverRepo } from "../../../repositories/OnlineDriverRepo";
-import logo from '../../../../public/logo.jpg';
+import logo from "../../../../public/logo.jpg";
+import { useRepository } from "../../../hooks/CustomHook";
```

Comment:
Let's set up "alias" in config files to shorten the import path to "@hooks/CustomHook".

Reply:
i added alias, but its not working as expected